src/ui/room_exits.ui

```diff
@@ -757,16 +757,62 @@
           </widget>
          </item>
          <item row="0" column="1">
-          <widget class="QLineEdit" name="roomID">
-           <property name="cursor">
-            <cursorShape>ForbiddenCursor</cursorShape>
-           </property>
-           <property name="toolTip">
-            <string>&lt;p&gt;This is the Room ID Number for this room - it cannot be changed here!</string>
-           </property>
-           <property name="readOnly">
-            <bool>true</bool>
-           </property>
+          <widget class="QFrame" name="frame_roomID">
+           <property name="frameShape">
+            <enum>QFrame::StyledPanel</enum>
+           </property>
+           <layout class="QHBoxLayout" name="horizontalLayout_roomID">
+            <property name="spacing">
+             <number>0</number>
+            </property>
+            <property name="leftMargin">
+             <number>0</number>
+            </property>
+            <property name="topMargin">
+             <number>0</number>
+            </property>
+            <property name="rightMargin">
+             <number>0</number>
+            </property>
+            <property name="bottomMargin">
+             <number>0</number>
+            </property>
+            <item>
+             <widget class="QLabel" name="roomID">
+              <property name="cursor">
+               <cursorShape>ForbiddenCursor</cursorShape>
+              </property>
+              <property name="toolTip">
+               <string>&lt;p&gt;This is the Room ID Number for this room - it cannot be changed here!</string>
```

Comment:
```suggestion
               <string>&lt;p&gt;Room ID for this room</string>
```

Saying 'it cannot be changed here!' adds no value - you already know that because the field doesn't let you. It is important to keep tooltips as consise as possible.